spec/classes/client_spec.rb

```diff
@@ -493,5 +493,25 @@
       end
     end
 
+    context 'where do_not_reset_file_access_time are set to invalid values' do
+      let :params do
+        {
+          :do_not_reset_file_access_time => 'foo',
+        }
+      end
+      it 'should fail' do
+        expect {
+          should contain_class('netbackup::client')
+          }.to raise_error(Puppet::Error)
+      end
+    end
+    context 'when use_ctime_for_incrementals set to valid true' do
+      let(:params) { { :use_ctime_for_incrementals => true } }
+      it { should contain_file('bp_config').with_content(%r{^USE_CTIME_FOR_INCREMENTALS = YES$}) }
+    end
+    context 'when use_ctime_for_incrementals set to valid undef' do
+      let(:params) { { :use_ctime_for_incrementals => 'undef' } }
+      it { should contain_file('bp_config').with_content(%r{^$}) }
```

Comment:
Good opportunity to be more specific:
`it { should_not contain_file('bp_config').with_content(%r{^USE_CTIME_FOR_INCREMENTALS.*$}) }`